bdk-ffi/src/wallet.rs

```
@@ -67,6 +67,48 @@ impl Wallet {
         })
     }
 
+    /// Build a new single descriptor `Wallet`.
+    ///
+    /// If you have previously created a wallet, use `Wallet::load` instead.
+    ///
+    /// # Note
+    ///
+    /// Only use this method when creating a wallet designed to be used with a single
+    /// descriptor and keychain. Otherwise the recommended way to construct a new wallet is
+    /// by using `Wallet::create`. It's worth noting that not all features are available
+    /// with single descriptor wallets, for example setting a `change_policy` on `TxBuilder`
```

> It's worth noting that not all features are available with single descriptor wallets, for example setting a `change_policy` on `TxBuilder` ...

are these just silent failures?